.github/workflows/build-and-deploy.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           PULUMI_STACK_NAME: ${{ vars.PULUMI_STACK_NAME }}
           DEPLOYMENT_ENVIRONMENT: ${{ vars.DEPLOYMENT_ENVIRONMENT }}
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-          GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
reverting this back to using GITHUB_TOKEN. The bot token is no longer needed since the PR workflow is now separated from this workflow and this one does not post previews as it is used only to deploy prod after merging.